src/proj.rs

```diff
@@ -595,6 +697,22 @@ mod test {
         assert_almost_eq(t.x(), 1450880.29);
         assert_almost_eq(t.y(), 1141263.01);
     }
+    // This test is disabled by default as it requires network access
+    // #[test]
+    // fn test_network() {
+    //     let from = "EPSG:4326";
+    //     let to = "EPSG:4326+3855";
+    //     // off by default
+    //     assert_eq!(network_enabled(), false);
+    //     // switch it on and disable cache for subsequent calls
+    //     grid_cache_set_enable(false);
+    //     enable_network(true).unwrap();
```

Comment:
😬 I feel like you're going to regret asking me to review this, because I have so many questions, but...

Can you explain what this is test testing? 

If I turn off my network, the test fails, which seems reasonable.

But I was surprised I could make it pass by applying this patch:
```diff
         assert_eq!(network_enabled(), false);
         // switch it on and disable cache for subsequent calls
         grid_cache_set_enable(false);
-        enable_network(true).unwrap();
+        // enable_network(true).unwrap();
         let proj = Proj::new_known_crs(&from, &to, None).unwrap();
-        assert_eq!(network_enabled(), true);
+        // assert_eq!(network_enabled(), true);
         let t = proj.convert(Point::new(40.0, -80.0)).unwrap();
         assert_almost_eq(t.x(), 39.99999839);
         assert_almost_eq(t.y(), -79.99999807);
```

So to clarify, it seems like here we're testing the network configuration - but none of the computations in the test actually require network, is that right?

Reply:
No, don't worry, I asked you because I knew this would happen!
You're absolutely correct, I just concocted a transform - the main idea was to trigger a download. I'll figure out something unambiguous that makes the test's intent clear.

Reply:
The new test (`test_network_enabled_conversion`) demonstrates different results for the same conversion operation, depending on whether a grid was downloaded by enabling network functionality.